latex2sympy.py

```diff
@@ -814,6 +827,24 @@ def apply_nested(f, lst):
     return sympy.UnevaluatedExpr(result)
 
 
+def handle_floor(expr):
+    """
+    Apply floor() then return the evaluated expression.
```

Comment:
@tirkarp since we are using `evaluate=False` these methods are not technically returning evaluated expressions

Reply:
okay. wrong terminology.